agent/hcp/telemetry/otlp_transform.go

```diff
@@ -96,7 +96,7 @@ func metricTypeToPB(m metricdata.Metrics) (*mpb.Metric, error) {
 		}
 	case metricdata.Sum[float64]:
 		if a.Temporality != metricdata.CumulativeTemporality {
-			return out, fmt.Errorf("error: %w: %T", temporalityErr, a)
+			return out, fmt.Errorf("error: %w: %T", errTemporality, a)
```

Comment:
nit, also applying to other errs below

```suggestion
			return out, fmt.Errorf("failed to convert metric to otel format: %w: %T", errTemporality, a)
```